pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -1860,7 +1860,7 @@ protected void unloadTopic(TopicName topicName, boolean authoritative) {
         validateTopicOwnership(topicName, authoritative);
         try {
             Topic topic = getTopicReference(topicName);
-            topic.close().get();
+            topic.close(false).get();
```

Comment:
I'd rather have an enum here because it's not clear what false means in this context.

Reply:
umm.. I think closing forcefully can be represent with boolean flag as we do similar thing at multiple places: `PersistentTopic::delete(boolean... flags)`

Also, I was also trying to think about how to accommodate enum here instead of flag. One thing I can think of is to  add below enum under `Topic` instead of flag.
```
enum CLOSE_ACTION { CLOSE_ALL, CLOSE_WITHOUT_CLIENT_WAIT }
```
But I feel enum is not helping much. Instead we can rename the flag to give more meaning `closeWithoutWaitingClientDisconnect`. 

So, for `PersistentTopic` if flag is enabled then broker skips waiting on client-disconnect and immediately closes managed-ledger before giving up bundle ownership. 
And for `NonPersistentTopic` just completes the close if flag is enabled.

Can you please let me know if I am missing anything while renaming flag instead making enum.? any thoughts?